Tried to shorten all sections (#269)
ITU-DevOps2026/ITU-minitwit: report/sections/02-3-logging.md

## Logging
MiniTwit uses a push-based logging model where Serilog directly sends logs from all replicas to Elasticsearch for centralized viewing. The default logging level is set to *Information* or higher. However, to reduce noise, we override this for Microsoft, AspNetCore, and EFCore, capturing only *Warning* levels and above. These logs use the *minitwit-logs* index prefix for easy identification.

### What our current logging implementation is missing

> Because our infrastructure runs in Docker containers, we should have deployed a log-shipper like Filebeat to collect and forward container logs to Elasticsearch.

Reviewer: ```suggestion
Because our infrastructure is containerized, we should have deployed a log-shipper like Filebeat to collect and forward container logs to Elasticsearch.
```

> Additionally, our logging statements in MiniTwit.cs are sparse, leaving some functions unlogged. While not everything requires a log, this coverage should be structured more deliberately than it is currently

Reviewer: ```suggestion
Additionally, our logging statements in MiniTwit.cs are sparse, leaving functions unlogged. While not everything requires a log, this coverage should be structured more deliberately than it is currently.
```
